lib/src/sidenav/Sidenav.tsx

```diff
@@ -1,56 +1,140 @@
-import React from "react";
+import React, { forwardRef, ReactElement, Ref, useMemo, useState } from "react";
 import styled, { ThemeProvider } from "styled-components";
-import { spaces, responsiveSizes } from "../common/variables.js";
+import { responsiveSizes } from "../common/variables.js";
+import { useResponsiveSidenavVisibility } from "../layout/SidenavContext";
 
 import useTheme from "../useTheme";
 import { BackgroundColorProvider } from "../BackgroundColorContext";
-import SidenavPropsType, { SidenavTitlePropsType, SidenavSubtitlePropsType, SidenavLinkPropsType } from "./types";
-import { useResponsiveSidenavVisibility } from "../layout/SidenavContext";
+import SidenavPropsType, {
+  SidenavGroupPropsType,
+  SidenavLinkPropsType,
+  SidenavSectionPropsType,
+  SidenavTitlePropsType,
+} from "./types.js";
+import DxcStack from "../stack/Stack";
 
-const DxcSidenav = ({ padding, children }: SidenavPropsType): JSX.Element => {
-  const colorsTheme = useTheme();
+const collapsableIcon = (
```

Comment:
The icons of a collapsable group are exchanged:

![image](https://user-images.githubusercontent.com/44321109/176678214-ffa5d1e6-7b36-48d7-9a7d-94d5a585dfe8.png)

This should be all the way around, as Alex had in the design and we are currently doing in the Select.

![image](https://user-images.githubusercontent.com/44321109/176678420-6ce121da-5e7b-46dd-9bad-0a65f9a0417c.png)